src/components/breadcrumbs/crumb/crumb.component.tsx

```diff
@@ -28,47 +28,32 @@ export interface CrumbProps
   isCurrent?: boolean;
 }
 
-function mergeRefs<T>(...refs: Array<React.Ref<T> | undefined>) {
-  return (node: T) => {
-    refs.forEach((ref) => {
-      if (!ref) return;
-      if (typeof ref === "function") {
-        ref(node);
-      } else {
-        (ref as React.MutableRefObject<T | null>).current = node;
-      }
-    });
-  };
-}
-
 const Crumb = React.forwardRef<HTMLElement, CrumbProps>(
   ({ href, isCurrent, children, onClick, ...rest }, forwardedRef) => {
     const { isDarkBackground } = useBreadcrumbsContext();
     const crumbRef = React.useRef<HTMLElement | null>(null);
 
+    React.useImperativeHandle(forwardedRef, () => crumbRef.current, []);
```

Comment:
Comment: this isn't how you use an imperative handle (I'm not convinced the TS types will allow it either way), I've already recommended an approach that will work here